tests/test_conditional.py

```diff
@@ -0,0 +1,76 @@
+from pytest import TempPathFactory
```

Comment:
Would you mind to please name the file `test_conditional_file_name.py`?

Just `test_conditional.py` is not much specific (we have a lot of "conditionals" to test).